src/manual/using/actions/schema.yml

```diff
@@ -18,160 +58,79 @@ properties:
-          description: |
-            JSON schema for input parameters to the `task` template property.
-            Consumers shall offer a user-interface where end-users can enter
-            values that satisfy this schema. Furthermore, consumers **must**
-            validate enter values against the given schema before parameterizing
-            the `task` template property and triggering the action.
-
-            In practice it's encourage that consumers employ a facility that
-            can generate HTML forms from JSON schemas. However, if certain
-            schemas are particularly complicated or common, consumers may also
-            hand-write a user-interface for collecting the input. In this case
-            the consumer **must** do a deep comparison between the schema given
-            in the action, and the schema for which a custom user-interface have
-            been written, and fall-back to an auto-generated form if the schema
-            doesn't match.
-
-            It is assumed that the JSON schema `description` property will be
-            rendered as markdown when displayed as documentation for end-users.
-            Producers of `public/actions.json` is encouraged to provide a
-            detailed explanation of the input parameters using these
-            `description` properties. And consumers are *strongly* encouraged
-            to render `description` values as markdown.
-
-            The `schema` property is optional, and if not given the input for
-            `task` template parameterization shall be `null`.
-        task:
-          type: object
-          title: task template
-          description: |
-            Task template for triggering the action.
-
-            When an action have been selected in the appropriate context and
-            input satisfying the `schema` (if any) has been collected. The
-            action is triggered by parameterizing the task template given in
-            this property, and creating the resulting task.
-
-            The template is an object that is parameterized using
-            [JSON-e](https://github.com/taskcluster/json-e), with the above
-            variables supplied as context.
+              Task template for triggering the action.
+
+              When an action have been selected in the appropriate context and
+              input satisfying the `schema` (if any) has been collected. The
+              action is triggered by parameterizing the task template given in
+              this property, and creating the resulting task.
+
+              The template is an object that is parameterized using
+              [JSON-e](https://github.com/taskcluster/json-e), with the context
+              described in the documentation.
+        additionalProperties: false
+        required:
+          - kind
+          - title
+          - description
+          - task
+      - type: object
+        properties:
+          kind: {enum: hook}
```

Comment:
This has to be `kind: {enum: ['hook']}`

Reply:
good point - I trusted https://github.com/taskcluster/taskcluster-docs/pull/250#discussion_r181486122 :)